src/style.css

```
@@ -0,0 +1,80 @@
+body {
+  margin: 0 8px;
+}
+
+.common_input {
+  font-family: "Avenir-book", sans-serif;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.common_input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.common_form {
+  width: 100%;
+}
+
+.big_form {
+  margin-top: 20px;
+}
+
+.small_form {
+  margin-top: 20px;
+}
```

you can combine selectors with the same styles
```suggestion
.big_form, .small_form {
  margin-top: 20px;
}
```